package.json

```diff
@@ -42,5 +41,9 @@
     "eslint-plugin-promise": "^3.8.0",
     "eslint-plugin-react": "^7.9.1",
     "eslint-plugin-standard": "^3.1.0",
+    "mini-css-extract-plugin": "^0.4.0",
```

Comment:
can you remove this one? since we are not using it anymore

Reply:
done.